tutorials/quickstart.eve.md

```diff
@@ -0,0 +1,92 @@
+# Quickstart
+
+In this guide we're going to give a 5 minute introduction to the essential concepts in Eve. If you've never used Eve before, you're in the right place.
+
+## Adding records to Eve
+
+Eve represents data as records, which are key value pairs attached to a unique ID. Eve programs are made up of small composable blocks that search for and create records. Let's start with a block that adds a record to Eve:
+
+```eve
+commit
+  [#greeting text: "hello world"]
+```
+
+This record is tagged `#greeting`, and has an attribute "text" with the value "hello world". Although tags have a special syntax, they are attributes like any other (i.e. #foo = tag: "foo"). We encourage you to classify groups of related records with tags.
+
+## Finding records in Eve
+
+We can search for records, then bind new records to those that are found. Eve finds every record matching the supplied patterns, and then binds records dependent on them. If no records match the search, then the block does not run at all. A block will only run if every pattern in a search matches at least one record. Let's search for the `#greeting` we just committed, and then display it in a text container:
+
+```eve
+search
+  [#greeting text]
+
+bind
+  [#ui/text text]
+```
+
+Variables with the same name are equivalent within a block; because they have the same name, the `text` in `[#greeting text]` and `[#ui/text text]` are equivalent. Go ahead and add another `#greeting` record to the first block to see what happens when more than one record matches the search (you can add it on a new line under the one that's there). 
```

Comment:
:+1: Agreed, solid explanation of aliasing. Instructions unclear about adding another greeting though. What did you expect me to do? All the things I'd normally think of wouldn't change the output at all.